.gitignore

```diff
@@ -330,3 +330,6 @@ ASALocalRun/
 
 # MFractors (Xamarin productivity tool) working folder
 .mfractor/
+
+# Do not account for changes in launchSettings.json
+**/Properties/launchSettings.json
```

Comment:
Apparently this had no effect

Reply:
The .gitignore file doesn't apply to files already tracked or explicitly added: [git help gitignore](https://git-scm.com/docs/gitignore)
> Files already tracked by Git are not affected